twisted/conch/ssh/keys.py

```diff
@@ -782,18 +783,23 @@ def public(self):
         return Key(self._keyObject.public_key())
 
 
-    def fingerprint(self):
+    def fingerprint(self, hash_function=sha256):
```

Comment:
the signature should be 
`def fingerprint(self, hashFunction=md5):
or 
`def fingerprint(self, digest=md5):`

we should not change the behaviour of exiting code using `fingerprint()` and hash_function does not comply with Twisted coding standard